.babelrc

```diff
@@ -0,0 +1,22 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
```

Comment:
@orballo does this `.babelrc` file affect the browsers we support in the plugin?

Reply:
@luisherranz That's actually somthing we should check. I don't know what support we are giving on the plugin. I was testing if the tree shaking was working with `corejs` and ended up like this just because. So if you tell me what support we should give I'll change the targets.